website/docs/getting-started/setup/installation-guides/aws-ami.md

```diff
@@ -1,145 +1,113 @@
+ <figure>
+ <img src="/img/aws-login-page.png" style={{width: "100%", height: "auto"}} alt="Appsmith Login Screen" />
+ <figcaption align="center"><i>Appsmith Login Screen</i></figcaption>
+ </figure>
 
-![EC2 Detail](/img/aws-ec2-detail.png)
+12. Follow these steps to log into the instance using admin credentials:
 
-At this point, you should be able to browse the cloud server, by entering the cloud server IP address or DNS name directly into your browser’s address bar. You should now see your web app home page as shown below:
+ a. Username: `appsmith@example.com`
 
-![Login Page](/img/aws-login-page.png)
+ b. Find the password in one of the below ways:
 
-## Application Credentials
-
-By default, Appsmith boots up with default user credentials that allow you to log in without needing to sign up. You can use the default username (`appsmith@example.com`). There are two options for obtaining the password.
-
-### Option 1: Find credentials by checking the System log on the AWS Cloud Console (EC2)
-
-:::caution Attention
-The application password is only available in the system logs for the first 24 hours after you first start the instance. It's recommended that you note it down immediately on the first boot and save it in a safe place, as you need it to access the instance console. It's also recommended that you change it as soon as possible for security reasons.
-:::
-
-* Go to your EC2 instances dashboard on AWS
-* Select the instance
-* From the “Actions” drop-down menu, select the “Get System Log” menu item.
-
-![Select System Log](/img/aws-select-system-log.png)
-
-* Review the system log until you find the application password. You can also find the default username printed in the logs.
-
-![View System log](/img/aws-system-log.png)
-
-### Option 2: Find credentials by connecting to your application through SSH
-
-The default application credentials are stored in a standalone file. To obtain these credentials at any time, follow these instructions:
-
-* SSH into your server using your private key
-*   Run the following command to see your application credentials:
-
-    ```
-      sudo cat /home/ubuntu/appsmith/credential
+    * SSH into your server using your private key and find the application credentials with:
     ```
+    sudo cat /home/ubuntu/appsmith/credential
+    ``` 
+    * **Or** review the system log, by selecting **Get System Log** from the **Actions** drop-down menu for your instance. The credentials are printed in the logs as shown below:
 
-## Updating your Appsmith installation
-
-You can either choose to update the Appsmith installation manually or choose to enable auto-updates.
+      :::caution Attention
+      The application password is **only** available in system logs for the initial 24 hours. It's recommended that you change the password for security reasons.
```

Comment:
I think the logs are rotated based on the size, are we sure it gets ceared in 24 hours? I'll check this out.

Reply:
This was available in existing docs. So kept it as is.
<img width="885" alt="image" src="https://github.com/appsmithorg/appsmith-docs/assets/99252011/b8f18349-55a8-410b-bbb3-a02f99140fc2">